test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/MetadataTests.cs

```diff
@@ -31,5 +32,13 @@ public async Task WhenGettingMetadata_GivenSystemFlag_TheServerShouldReturnSucce
 
             Assert.NotEmpty(readAsync.Resource.Rest);
         }
+
+        [Fact]
+        [Trait(Traits.Priority, Priority.One)]
+        public async Task WhenGettingMetadata_GivenInvalidFormatParameter_TheServerShouldThrowNotAcceptableException()
```

Comment:
nit: `TheServerShouldReturnNotAcceptable` ?

Reply:
Thanks, fixed in d326d90.